src/language-js/print/binaryish.js

```diff
@@ -267,13 +281,28 @@ function printBinaryishExpressions(
           "right",
         )
       : print("right");
-    right = [
-      lineBeforeOperator ? line : "",
-      operator,
-      lineBeforeOperator ? " " : line,
-      rightContent,
-      rightSuffix,
-    ];
+    if (options.experimentalOperatorPosition) {
+      let comment = "";
+      if (commentBeforeOperator) {
+        switch (getDocType(rightContent)) {
+          case DOC_TYPE_ARRAY:
+            comment = rightContent.splice(0, 1)[0];
+            break;
+          case DOC_TYPE_LABEL:
+            comment = rightContent.contents.splice(0, 1)[0];
+            break;
+        }
+      }
+      right = [line, comment, operator, " ", rightContent, rightSuffix];
+    } else {
+      right = [
```

Comment:
This assignment to `right` is just an indentation change from moving it into the `else` branch. Hide whitespace to remove this from the diff.